float32 narrow_l1
float32 narrow_l2
float32 narrow_l3
float32 narrow_r1
float32 narrow_r2
float32 narrow_r3
float32 forward
float32 left
float32 right
float32 back
float32 closest_dist
string closest_dir
float32 norm_bearing
float32 norm_dist

